<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PharmacistGroupBox</class>
 <widget class="QGroupBox" name="PharmacistGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="title">
   <string notr="true">Pharmacist Information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="lastTitle">
       <property name="toolTip">
        <string notr="true">Last name</string>
       </property>
       <property name="text">
        <string notr="true">Last name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lastLabel">
       <property name="toolTip">
        <string notr="true">Last name</string>
       </property>
       <property name="text">
        <string notr="true">PH LAST</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="firstTitle">
       <property name="toolTip">
        <string notr="true">First name</string>
       </property>
       <property name="text">
        <string notr="true">First name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="firstLabel">
       <property name="toolTip">
        <string notr="true">First name</string>
       </property>
       <property name="text">
        <string notr="true">PH FIRST</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="initialsTitle">
       <property name="toolTip">
        <string notr="true">Initials</string>
       </property>
       <property name="text">
        <string notr="true">Initials:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="initialsLabel">
       <property name="toolTip">
        <string notr="true">Initials</string>
       </property>
       <property name="text">
        <string notr="true">PH INITIALS</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="activeTitle">
       <property name="toolTip">
        <string notr="true">Active or inactive status</string>
       </property>
       <property name="text">
        <string notr="true">Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="activeLabel">
       <property name="toolTip">
        <string notr="true">Active or inactive status</string>
       </property>
       <property name="text">
        <string notr="true">PH ACTIVE</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
